alchemist/src/orchid/resources/wiki/usage/run.md

```diff
@@ -0,0 +1,46 @@
+---
+
+title: "Running Alchemist simulations"
+
+---
+
+Alchemist expects a YAML map as input, such maps are contained in *.yml files and are used to write simulations. Informations about how to write alchemist simulations can be found in the next section.
+
+## Running a simulation
+
+The typical way of running alchemist simulations is by directly using alchemist <kbd>main</kbd> method. If you have cloned the Alchemist repository, you can also run a simulation using the produced artifacts (namely, a .jar file). The first approach is recommended.
+
+**Running a simulation using Alchemist main**
+
+Alchemist main class (i.e. the class which contains the <kbd>main</kbd> method) is the following: <kbd>it.unibo.alchemist.Alchemist</kbd>. In order to run a simulation, you can call its main method passing proper parameters (passing '-h' option will print out the supported options, to run a simulation you can rely on the <kbd>-y</kbd> option followed by the path to your own *.yml file).
```

Comment:
do not use `<kbd>` for highlighting code. use backticks: `` `main` ``: `main`
See: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

Reply:
Removed all `<kbd>` tags